InstituteBudget.php

```diff
@@ -387,43 +432,43 @@ public function getCCSummaryData() {
 	}
 	
 	public function showStaticScheduleArms($budget_data, $arms_and_visits_survey_link) {
-		foreach($budget_data->arms as $arm_i => $arm) {
+		foreach($budget_data['arms'] as $arm_i => $arm) {
 		$link_arm_index = $arm_i + 1;
 		?>
 		<div class="budget_container">
-		<a href="<?= $arms_and_visits_survey_link . "&arm=$link_arm_index" ?>"><h6><?= "Arm " . ($arm_i + 1) . ": " . ($arm->name ?? "") ?></h6></a>
+		<a href="<?= $arms_and_visits_survey_link . "&arm=$link_arm_index" ?>"><h6><?= "Arm " . ($arm_i + 1) . ": " . ($arm['name'] ?? "") ?></h6></a>
```

Comment:
consider using a private URL builder method

Reply:
I'll probably add something like this once I'm done cleaning out most of the unused code and moving some of this html into twig templates.